content/posts/0.19-release.md

```diff
@@ -0,0 +1,46 @@
+Pyodide version 0.19.0 has fewer changes to the core Pyodide runtime than
+previous versions. We focused on improving the build system and the systems for
+loading Python code. There is a lot more work to be done in this area.
+
+We hope to encourage people to develop their Python code for use in Pyodide like
+normal Python packages and to call into their Python packages directly from
+Javascript. Using `pyodide.runPython` directly is like using `eval`: it works
+alright in some cases, but usually there are better alternatives.
+
+The core type conversions system and Pyodide runtime remained much more stable
+than in the last few releases. We added polish to core features based on user
+feedback, including bugfixes and convenience improvements. Hopefully as
+we make fewer changes to core systems we will see fewer new problems come up.
```

Comment:
I'm going to shorten these sections a bit, as I think we don't need to compare with previous releases (most users won't care about that part IMO) but only indicate what was done in this release, particularly in the introduction.